http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -823,39 +841,41 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
             projectLabels <- googleResourceService.getLabels(googleProject)
             isAppAllowed <- projectLabels match {
               case Some(labels) =>
+                // as long as the project has SECURITY_GROUP label, it is an AoU project
```

Comment:
👍

Reply:
Not within the scope of this PR, but it would be nice to have a more explicit label to check if a project is an AOU one (similar to the aou label in the leo db for runtimes)

Reply:
maybe..we can definitely add that if it becomes a pattern. Btw, where do you see the aou label for runtimes?

Reply:
I am fairly sure that the label table in leo DB differenciates between the terra (`terraLabel`) and aou (`allOfUsLabel`) resources but that might be for the UI definition of a runtime, not the leo one 🤔

Reply:
ah right...UI sets some labels to differentiate terra vs AoU as well...I guess that could be an alternative solution too...I'll look into that

Reply:
@LizBaldo updated things to use label to differentiate..lmk if u have more comments

Reply:
LGTM, maybe you want to update the comment above to reflect the use of the UI label?

Reply:
I removed the comment for now...I think in reality, the statement is true today, but the comment is no longer super related to the code anymore